metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
PS: isn't dhparams the responsibility of the lets-encrypt layer? I'd think any ssl connection needs those, no? Does the letsencrypt layer generate those and if not, should it?